src/unix/udp.c

```diff
@@ -263,15 +263,11 @@ static void uv__udp_recvmsg(uv_udp_t* handle) {
 
 #if HAVE_MMSG
     uv_once(&once, uv__udp_mmsg_init);
-    if (uv__recvmmsg_avail) {
-      /* Returned space for more than 1 datagram, use it to receive
-       * multiple datagrams. */
-      if (buf.len >= 2 * UV__UDP_DGRAM_MAXSIZE) {
-        nread = uv__udp_recvmmsg(handle, &buf);
-        if (nread > 0)
-          count -= nread;
-        continue;
-      }
+    if (uv__recvmmsg_avail && handle->flags & UV_HANDLE_UDP_RECVMMSG) {
```

Comment:
```suggestion
    if (uv__recvmmsg_avail && (handle->flags & UV_HANDLE_UDP_RECVMMSG)) {
```

Reply:
Somehow half of my comment disappeared...

I suggested moving the `handle->flags & UV_HANDLE_UDP_RECVMMSG` check up so that it encloses the `uv_once()` call. That way you only pay for it when you ask for it.

Reply:
Makes sense. I was also thinking about making a follow up PR which does that check on loop creation. WDYT?

Reply:
No strong opinion, really. Either way is fine.